ablateLibrary/particles/inertial.hpp

```diff
@@ -1,5 +1,6 @@
 #ifndef ABLATELIBRARY_INERTIAL_HPP
 #define ABLATELIBRARY_INERTIAL_HPP
+#include "drag/dragModel.hpp"
```

Comment:
I don't think that the dragModel is being used yet in inertial, I would remove it for now.

Reply:
I removed this for now, will add later as appropriate.